src/styles/variables.js

```diff
@@ -14,9 +14,11 @@ export default {
     fontSizeLabel: 13,
     fontSizeNormal: 15,
     fontSizeLarge: 17,
+    fontSizeExtraLarge: 36,
```

Comment:
I think a more accurate description for this might be `fontSizeHero` in that it's the large font size we might use for big hero moments.

Reply:
Done